src/main/kotlin/uk/gov/dluhc/printapi/database/entity/TemporaryCertificateStatus.kt

```diff
@@ -0,0 +1,71 @@
+package uk.gov.dluhc.printapi.database.entity
+
+import org.hibernate.Hibernate
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.Type
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import uk.gov.dluhc.printapi.database.repository.UUIDCharType
+import uk.gov.dluhc.printapi.database.repository.UseExistingOrGenerateUUID
+import java.time.Instant
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
+import javax.persistence.Version
+import javax.validation.constraints.NotNull
+import javax.validation.constraints.Size
+
+@Table
+@Entity
+@EntityListeners(AuditingEntityListener::class)
+class TemporaryCertificateStatus(
+    @Id
+    @Type(type = UUIDCharType)
+    @GeneratedValue(strategy = GenerationType.IDENTITY, generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = UseExistingOrGenerateUUID.NAME)
+    var id: UUID? = null,
+
+    @field:NotNull
+    @Enumerated(EnumType.STRING)
+    var status: Status? = null,
+
+    @field:NotNull
+    @field:Size(max = 255)
+    var userId: String? = null,
+
+    @CreationTimestamp
+    var dateCreated: Instant? = null,
+
+    @field:Size(max = 255)
+    @LastModifiedBy
+    var createdBy: String? = null,
+
+    @Version
+    var version: Long? = null
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (other == null || Hibernate.getClass(this) != Hibernate.getClass(other)) return false
+        other as TemporaryCertificateStatus
+
+        return id != null && id == other.id
+    }
+
+    override fun hashCode(): Int = javaClass.hashCode()
+
+    @Override
+    override fun toString(): String {
+        return this::class.simpleName + "(id = $id , dateCreated = $dateCreated , createdBy = $createdBy , version = $version )"
+    }
+
+    enum class Status {
```

Comment:
I've added the `Status` enum as an inner class. This is because there is already a `Status` enum in the entity package, which is used for the main certificate.
The values in these 2 status enums are different; and we need to be able to differentiate between the 2, so an inner class felt right.

Once this PR is merged then I'm planning on refactoring the existing `Status` enum (that is used for regular certificates) as an inner class of `PrintRequestStatus` - but I'll do that separately to this PR because otherwise this would be massive and impossible to follow